interface/billing/sl_eob_search.php

```diff
@@ -615,7 +615,7 @@ function npopup(pid) {
 
    <table border='0' cellpadding='5' cellspacing='0'>
 
-     <tr bgcolor='#ddddff'>
+     <tr class = "tabs_body_light" bgcolor='#ddddff'>
```

Comment:
These are used for styling in flat stylesheets without affecting other stylesheets